pages/ugm2024.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: UGM 2024
+---
+
+## When, Where, and How
+The UGM will be held in a live, in-person format across 3 days, August 21-23, 2024, in London, UK ([Bush House](https://www.kcl.ac.uk/visit/bush-house) at King's College London).
+
+## What to Expect
+The UGM will bring together users of the MDAnalysis package from different communities. Our goal is to foster an opportunity to connect for interdisciplinary researchers and developers across biomolecular simulations, soft matter, biophysics and more. Whether you are a new MDAnalysis contributor, an MDAnalysis core developer, or a researcher using (or interested in using) MDAnalysis for your work in academia or industry, this UGM is for you. You can expect:
+* Keynote talks from experts in the molecular dynamics simulations space (see [Keynote Speakers](https://www.mdanalysis.org/pages/ugm2023/#keynote-speakers) below)
+* Talks related to developing and using MDAnalysis for diverse scientific applications
+* Tutorials diving into new features
+* Direct access to the [MDAnalysis core developer team][coredev] to help shape future development of MDAnalysis
+* Networking opportunities to foster new collaborations
+* Hackathon and other social outings
+
+## Keynote Speakers
+Keynote speakers will be announced shortly.
+
+## Program
+The agenda will be announced shortly.
+
+## Registration and Abstracts
+Registration details and a call for abstracts will be announced shortly.
```

Comment:
we need to give ourselves a deadline. all we need to do here is
- copy paste a new eventbrite, 
- figure out whether we want to charge people for the social event 
- organise the event 
so I propose March 1st or April 1st as the registration start

Reply:
I agree, just put a date and make it work.

Reply:
Also, as just discussed at the call, if we have a two-step process of
1. abstract submission
2. meeting registration

then we should make the process clear.

Reply:
How about 
- Feb 1st opening for abstract submission 
- Deadline end of March for abstracts
- Registration + eventual payment opens mid-March?

Reply:
These suggested dates work for me. I have added them in, as well as added language for the bursary applications.

Reply:
that's ok by me!